src/components/common/form/PercentField.js

```diff
@@ -4,6 +4,12 @@ import { Form, Input } from 'semantic-ui-react'
 import { InputRef } from './InputRef'
 import ErrorMessage from './ErrorMessage'
 
+export function moveDecimalsRight(inputNum) {
+  let numAsStr = inputNum.toString()
+  let numAfterDecimalsStr = numAsStr.substring(2, 4)
```

Comment:
@micheal-w-wells This function doesn't always work as expected. For values like `0.2`, this function will return `2`, when we want it to return `"20"`. This also breaks the `allowDecimals` functionality, which lets the consumer of this component choose if the field should allow decimals (eg. `"20.44%"`) or not.